test/e2e/specs/onboarding/ftme__site-assembler.ts

```diff
@@ -113,7 +113,7 @@ describe( 'Site Assembler', () => {
 		} );
 
 		it( 'Pick default style', async function () {
-			await siteAssemblerFlow.clickButton( 'Pick your style' );
```

Comment:
@arthur791004 do you know why does this test pass? IIRC we changed it to `Pick your styles` last week, and the hook `hasEnTranslations` should return true in these E2E test. Am I missing something? 🤔

Reply:
The underlying [implementation](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-e2e/src/lib/flows/site-assembler-flow.ts#L25) of `clickButton` uses regex, so that's why it still passes.

Reply:
Ah... I totally missed that. Thanks!